### PR TITLE
Add documentation to primary `funloc` example

### PR DESCRIPTION
**Pull request by wronk**
Bolster comments for bits I didn't understand at first

### CODE REVIEW

File: examples/funloc/analysis_fun.py [hunk 1]
```diff
@@ -95,24 +102,25 @@
 # Set what will run
 mnefun.do_processing(
     params,
-    fetch_raw=True,  # Fetch raw recording files from acq machine
-    do_score=True,  # do scoring
+    fetch_raw=True,     # Fetch raw recording files from acquisition machine
+    do_score=True,      # Do scoring to slice data into trials
     # Make SUBJ/raw_fif/SUBJ_prebad.txt file with space-separated
     # list of bad MEG channel numbers, needed for running SSS.
```
Comment (wronk) [inline examples/funloc/analysis_fun.py]: @Eric89GXL, do these (and some of the other full line comments below) belong? I'm not exactly sure what they're supposed to convey.

Comment (Eric89GXL) [inline examples/funloc/analysis_fun.py]: They should be done *before* doing the steps that follow

Comment (Eric89GXL) [inline examples/funloc/analysis_fun.py]: (done manually by the user)

**Comment by wronk** [comment]
Okay @Eric89GXL, ready for review

**Comment by Eric89GXL** [comment]
Thanks @wronk

**Pull request merged**